pyzeebe/channel/utils.py

```diff
@@ -0,0 +1,11 @@
+import os
+from typing import Optional
+
+
+def create_address(
+    hostname: Optional[str] = None,
+    port: Optional[int] = None,
+) -> str:
+    if hostname or port:
+        return f"{hostname or 'localhost'}:{port or 26500}"
+    return os.getenv("ZEEBE_ADDRESS", "localhost:26500")
```

Comment:
Can we move `localhost:26500` and `26500` to constants - to make clearer why these are used. 

```py
DEFAULT_HOSTNAME = "localhost"
DEFAULT_PORT = 26500
DEFAULT_ADDRESS = f"{DEFAULT_HOSTNAME}:{DEFAULT_PORT}"
```

If there are other references under `pyzeebe/`, these should maybe be imported from one file (single source of truth).